noo_dev_ocp.md

```diff
@@ -0,0 +1,55 @@
+# Developing network Operator (on OCP)
+
+The following instructions are useful for local speedy development of the operator.
+
+## prerequisites
+
+1. Local linux machine
+2. OCP cluster
+
+## Deployment process
+
+1. Connect to the OCP cluster using `oc login ...`
+2. execute `kubectl create namespace network-observability` - this creates the `network-observability` namespace
+3. execute `kubectl config set-context --current --namespace network-observability` - this changes the `kubectl` context to `network-observability` namespace
+4. Execute `./hack/deploy-loki.sh` - this  deploys a simple loki instance into the cluster
+4. Execute `./hack/deploy-grafana.sh` - this deploys a simple grafana instance into the cluster (user: admin password: admin)
```

Comment:
Do you plan to add some shells in this repo ? 

Here we use https://github.com/netobserv/documents/blob/main/loki_simple.md#hands-on
```
oc apply -f examples/zero-click-loki/1-storage.yaml
oc apply -f examples/zero-click-loki/2-loki.yaml
```

Reply:
I changed the #PR https://github.com/netobserv/network-observability-operator/pull/61 so now this is changed to using the commands `make deploy-loki` (which behind the scenes using those yaml files)

:-)